tests/dummy/app/docs/features/services/template.md

```diff
@@ -1,15 +1,5 @@
 # Using Services
 
-Services can be used through the [`@ember-decorators`][ember-decorators] addon. Start off by installing it
-
-```bash
-ember install ember-decorators@next
-```
-
-With that set up, you can inject references to a service the same way you can with an Ember component
+You can inject references to a service the same way you can with an Ember component
```

Comment:
I believe these caveats are outdated, given the addon would no longer support Ember v3.1, so I removed them.